public/scss/app/_desktop.scss

```diff
@@ -39,6 +39,16 @@ body {
 
     &.card {
       width: 100%;
+
+      &.files {
+        height: 60vh;
```

Comment:
Giving it a fixed height of 60vh makes it look weird when there's only 1 file in the list.

![issue](https://user-images.githubusercontent.com/21107799/66715011-c2015a00-eddb-11e9-9773-3bc40d77b8b7.png)

Reply:
Now the height will be dynamic and it will grow depending on the number of files, up to a point where the container will be scrollable

Reply:
Perfect!